spec/index.bs

```diff
@@ -1678,23 +1793,19 @@ To get a {{GPUAdapter}}, use {{GPU/requestAdapter()}}.
 <script type=idl>
 [Exposed=(Window, DedicatedWorker), SecureContext]
 interface GPUAdapter {
-    readonly attribute DOMString name;
     [SameObject] readonly attribute GPUSupportedFeatures features;
     [SameObject] readonly attribute GPUSupportedLimits limits;
     readonly attribute boolean isFallbackAdapter;
+    readonly attribute GPUAdapterInfo info;
 
     Promise<GPUDevice> requestDevice(optional GPUDeviceDescriptor descriptor = {});
+    Promise<GPUAdapterInfo> requestUnmaskedAdapterInfo(sequence<DOMString> hints);
```

Comment:
Define an enum of possible hints instead of DOMString?

Reply:
That creates a pattern where it's difficult to extend this information in the future because enums can't be feature tested and passing an unknown enum would reject with an error. An alternative is to make it always reject on any hint that the user agent does not wish to expose for consistency, which is what NavigatorUAData.getHighEntropyValues() does, but that feels developer hostile.

Reply:
Good point, I think keeping DOMString is the right choice.